src/toM3u8.js

```diff
@@ -30,14 +36,22 @@ const mergeDiscontiguousPlaylists = playlists => {
     } else {
       // first Period
       acc[name] = playlist;
+      acc[name].attributes.timelineStarts = [];
```

Comment:
seems more logical for this code to be the first if statement and the else to be for "Periods after first"